.github/workflows/lint.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Black Check
         uses: psf/black@stable
         with:
-          version: "22.3.0"
+          version: "latest"
```

Comment:
you need to change the precommit config in the same way

Reply:
It still isn't working but I am on it, thanks for the approve.